README.md

```diff
@@ -25,42 +27,84 @@ Prior to running this role via Packer, you need to make sure Ansible is installe
       }
     ],
 
+```
+
 The files should contain, at a minimum:
 
 **scripts/ansible.sh**:
 
-    #!/bin/bash -eux
-    # Install Ansible repository and Ansible.
-    apt-add-repository ppa:rquillo/ansible
-    apt-get update
-    apt-get install ansible
+An example for Ubuntu 16.04
+
+```bash
+#!/bin/bash -eux
+# Install Ansible repository and Ansible.
+apt -y install software-properties-common
+apt-add-repository ppa:ansible/ansible
+apt-get update
+apt-get install ansible
+```
+
+An example for Debian 8.8
+
+```bash
+#!/bin/bash -eux
+
+# Install Ansible repository and Ansible
+apt -y install software-properties-common
+echo "deb http://ppa.launchpad.net/ansible/ansible/ubuntu trusty main" | tee -a /etc/apt/sources.list
+apt-key adv --keyserver keyserver.ubuntu.com --recv-keys 93C4A3FD7BB9C367
+apt -y update
+apt -y install ansible
+```
+
 
 **ansible/main.yml**:
 
-    ---
-    - hosts: all
-      sudo: yes
-      gather_facts: yes
-      roles:
-        - geerlingguy.packer-debian
+```
+---
+- hosts: all
+  sudo: yes
+  gather_facts: yes
+  roles:
+    - geerlingguy.packer-debian
+```
+
 
 You might also want to add another shell provisioner to run cleanup, erasing free space using `dd`, but this is not required (it will just save a little disk space in the Packer-produced .box file).
 
 If you'd like to add additional roles, make sure you add them to the `role_paths` array in the template .json file, and then you can include them in `main.yml` as you normally would. The Ansible configuration will be run over a local connection from within the Linux environment, so all relevant files need to be copied over to the VM; configuratin for this is in the template .json file. Read more: [Ansible Local Provisioner](http://www.packer.io/docs/provisioners/ansible-local.html).
 
 ## Role Variables
 
-None.
+### vmware_install_open_vm_tools: "false"
```

Comment:
For my role variables explanations, I like to have the variable itself as a code snippet, and then the explanation below—e.g. https://github.com/geerlingguy/ansible-role-drush/blob/master/README.md#role-variables